modules/payment/src/Plugin/Commerce/PaymentGateway/PaymentGatewayBase.php

```diff
@@ -11,12 +11,16 @@
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
 use Drupal\Core\Plugin\PluginBase;
+use Drupal\Core\Plugin\PluginWithFormsInterface;
```

Comment:
PaymentGatewayInterface already extends PluginWithFormsInterface, you don't need to add it here.